src/Umbraco.Cms.Integrations.SEO.SemrushTools/Umbraco.Cms.Integrations.SEO.SemrushTools/App_Plugins/UmbracoCms.Integrations/SEO/SemrushTools/package.manifest

```diff
@@ -0,0 +1,7 @@
+﻿{
```

Comment:
Perhaps can just rename the folder "SemrushTools" to "Semrush"? Not a big deal of course, but I can see why you've used this name to be analogous with "CommerceTools", but in that case it's actually the name of the partner service. In this case it's just "Semrush".

Reply:
Item addressed.